certified-connectors/RamQuest Actions - UAT/apiProperties.json

```diff
@@ -0,0 +1,11 @@
+{
+    "properties": {
+        "connectionParameters": {},
+        "iconBrandColor": "#007ee5",

```

Comment:
please make sure you are using the iconBrandColor for your connector. Icon color must contain a non-transparent, non-white color (#ffffff) background and not-default color (#007ee5) that matches your specified icon background color. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector).